providers/fireflies/handlers.go

```diff
@@ -246,3 +247,246 @@ func (c *Connector) parseReadResponse(
 		params.Fields,
 	)
 }
+
+// nolint:gocognit,cyclop,funlen
+func (c *Connector) buildWriteRequest(
+	ctx context.Context, params common.WriteParams,
+) (*http.Request, error) {
+	url, err := urlbuilder.New(c.ProviderInfo().BaseURL)
+	if err != nil {
+		return nil, fmt.Errorf("failed to build URL: %w", err)
+	}
+
+	recordData, err := common.RecordDataToMap(params.RecordData)
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert record data to map: %w", err)
+	}
+
+	var mutation string
+
+	switch params.ObjectName {
+	case objectNameLiveMeeting:
+		if params.RecordId == "" {
+			mutation = getMutation("graphql/mutation_meeting.graphql", "liveMeeting", recordData)
+		} else {
+			return nil, ErrUpdateMeetingLinkNotSupported
+		}
+	case objectNameCreateBite:
+		if params.RecordId == "" {
```

Comment:
minor, but would be much easier to read if we extracted each case into it's own function.

```
switch {
case objectMeeting:
  return handleMeeting()
...
}
```
```